public/cli/lib/commands/convert.ts

```diff
@@ -47,12 +46,30 @@ export async function convert(
       }
       onProgress({ status: 'Combining video frames...', percent });
       await combineFrames({ audioFiles, imagesPath, framerateIn: 15, outputName });
-      // videoPathsToCombine.push(path.join(args.project.fullPath, args.project.name, book.name, chapter.chapter));
+      fs.rmdirSync(imagesPath);
+      if (isCombined) {
+        videoPathsToCombine.push(outputName);
+      }
     }
   }
-  if (isCombined) {
-    // await combineVideos(videoPathsToCombine, outputFilePath);
+  if (videoPathsToCombine.length > 0) {
+    await combineVideos(videoPathsToCombine, outputFilePath);
+    fs.rmdirSync(path.dirname(videoPathsToCombine[0]));
```

Comment:
This line only deletes the directory of the first video. But `getOutputFilePath` creates a different temp directory for every video file.

Reply:
Done. I've passed in the combine directory to ensure they are the same and so we only have one to remove.
I've also moved the creation of the tmp director for rendering frames inside the chapter for loop (since we also remove it there afterward) so that we only combine frames from each chapter.

Reply:
Ok, turns out `rmdirSync` only removes an empty directory. Otherwise it throws a `ENOTEMPTY: directory not empty` error.
Try using `rmSync`

Reply:
Thanks, that should have worked but gave a different error about not being a function. I have used `tempy.directory.task` which deletes when the callback is done. I was hoping I could avoid using this but seems that is what it was made for.